apps/bfd-server/bfd-server-launcher/src/main/java/gov/cms/bfd/server/launcher/DataServerLauncherApp.java

```diff
@@ -81,6 +81,18 @@ public final class DataServerLauncherApp {
   public static final String HTTP_ACCESS_RESPONSE_OUTPUT_SIZE_IN_BYTES =
       "http_access_response_output_size_in_bytes";
 
+  /** MDC key for the http response duration per kb. */
+  public static final String HTTP_ACCESS_RESPONSE_DURATION_PER_KB =
+      "http_access_response_duration_per_kb";
+
+  /** MDC key for the http response duration milliseconds. */
+  public static final String HTTP_ACCESS_RESPONSE_DURATION_MILLISECONDS =
+      "http_access_response_duration_milliseconds";
+
+  /** MDC key for the http response start in milliseconds. */
+  public static final String HTTP_ACCESS_RESPONSE_START_MILLISECONDS =
+      "http_access_response_start_milliseconds";
+
```

Comment:
lets move these out of here and into BfdMdc as constants (you should be able to use the BfdMdc.compute function when you do that as well) that can be used here and in the RequestResponseMdcFilter so that we don't have to synchronize the names across classes.

Reply:
done